src/Pos/Types/Arbitrary.hs

```diff
@@ -42,14 +42,8 @@ import           Pos.Constants              (epochSlots, sharedSeedLength)
 import           Pos.Core.Address           (makePubKeyAddress, makeRedeemAddress,
                                              makeScriptAddress)
 import           Pos.Core.Coin              (coinToInteger, divCoin, unsafeSubCoin)
-import           Pos.Core.Types             (Address (..), ChainDifficulty (..), Coin,
-                                             CoinPortion, EpochIndex (..),
-                                             EpochOrSlot (..), LocalSlotIndex (..),
-                                             SharedSeed (..), SlotId (..), Timestamp (..),
-                                             getCoinPortion, mkCoin,
-                                             unsafeCoinPortionFromDouble, unsafeGetCoin)
-import           Pos.Core.Types             (ApplicationName (..), BlockVersion (..),
-                                             SoftwareVersion (..))
+import qualified Pos.Core.Types             as Types
```

Comment:
@gromakovsky On one hand, the imports for `Pos.Core.Types` were nearing 30 individual types and functions, I think it's worth it to make it qualified. On the other, having a separate import for `Coin` will keep `hlint` in perpetual nagging regarding double import. The latter is preferable, but not strictly necessary.